CHANGELOG.md

```diff
@@ -13,11 +13,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Requirements validation during `snforge` runtime
 - `snforge check-requirements` command
+- `snforge new` command
 
 #### Changed
 
  - `snforge_scarb_plugin` will now also emit warnings when errors occur
 
+#### Deprecated
+
+- `snforge init` command has been deprecated
+
```

Comment:
I think we should just put it under `Changed`, `Added`, `Changed` or `Fixed` are the ones we always use.

Reply:
From our changelog: "The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)"

From the `Keep a Changelog`:
```
Types of changes
Added for new features.
Changed for changes in existing functionality.
Deprecated for soon-to-be removed features.
Removed for now removed features.
Fixed for any bug fixes.
Security in case of vulnerabilities.
```
https://keepachangelog.com/en/1.0.0/#how

Reply:
Ok, I take it back 👍